include/ppx/grfx/grfx_command.h

```diff
@@ -582,6 +582,15 @@ class CommandBuffer
     //
     void Draw(const grfx::FullscreenQuad* pQuad, uint32_t setCount, const grfx::DescriptorSet* const* ppSets);
 
+    bool HasActiveRenderPass() const;
+
+protected:
+    bool                           mDynamicRenderPassActive = false;
+    grfx::Rect                     mRenderArea              = {};
+    grfx::RenderTargetView* const* mRenderTargetView        = {};
+    uint32_t                       mRenderTargetCount       = 0;
```

Comment:
Would you be able to either: 
- document that these are values for dynamic render passes only
- or save those on BeginRenderPass as well, which would potentially allow unifying code branches in ClearRenderTarget

Reply:
Done.